packages/currency/src/currencyManager.ts

```diff
@@ -71,17 +71,21 @@ export class CurrencyManager<TMeta = unknown> implements ICurrencyManager<TMeta>
       return this.fromAddress(currencyIdentifier, network);
     }
 
+    if (network && currencyIdentifier.indexOf(network) === -1) {
+      currencyIdentifier = `${currencyIdentifier}-${network}`;
+    }
+
+    const currencyFromId = this.fromId(currencyIdentifier);
+
+    if (currencyFromId) return currencyFromId;
```

Comment:
I don't get this. If you are using this method by passing always IDs, why not use `fromId` directly?

Reply:
You're right, we should always use `fromId` in user code when we are sure to have an ID as input.

In the case of `from`, we're not entirely sure what kind of input we have (ID, symbol, address). In this case, we want to prioritize looking by ID rather than symbol because it's more precise (like we already prioritize looking by address first). More context here: https://github.com/RequestNetwork/requestNetwork/pull/1345#issuecomment-1921277612

I'm sure we are over-using `from` sometimes where we could replace it with `fromId` in user code, but the path with `from` should work too and needs to be fixed.

Reply:
should we deprecate `from`?

Reply:
Yes, that could be a good start 👍 adding the `@deprecated` JSDOC